src/EulerHS/Framework/Flow/Interpreter.hs

```diff
@@ -282,6 +282,19 @@ interpretFlowMethod _ _ (L.GenerateGUID next) = do
 interpretFlowMethod _ _ (L.RunSysCmd cmd next) =
   next <$> readCreateProcess (shell cmd) ""
 
+interpretFlowMethod _ _ (L.ResolveDNS host next) = do
+  pure $ next (Map.lookup host dnsDatabase)
+  where
+    getTLD :: Text -> DNS -> Maybe Text
+    getTLD domainName dns = Map.lookup domainName dns
+
+    dnsDatabase :: DNS
```

Comment:
https://hackage.haskell.org/package/dns-4.2.0/docs/Network-DNS-Resolver.html#v:defaultResolvConf

use this /etc/resolv.conf info along with these hardcoded ones.